libs/design/src/molecules/loading-icon/loading-icon.component.scss

```diff
@@ -1,3 +0,0 @@
-i {
-  padding: 40px calc(50% - 37px);
```

Comment:
I think you either still need this, or you need to add styles to all of its instances in foundation-demo, because right now all of the loading icons are left aligned

Reply:
resolved in https://github.com/graycoreio/daffodil/pull/180/commits/a13ce8784f289a719e0f1e2078273cc4981da77c#diff-e98b7d880b148af142b83cf3e2c85cbdR19